packages/block-library/src/cover/style.scss

```diff
@@ -50,6 +50,10 @@
 		background-color: $black;
 	}
 
+	> .has-background-dim.has-background-gradient {
+		background-color: transparent;
+	}
+
```

Comment:
This is working well for me on trunk. Thanks for the patch!

I'm looking at the [deprecations](https://github.com/WordPress/gutenberg/blob/4bfd7b3d8a9ff575ccae27ee690ccd7327811574/packages/block-library/src/cover/deprecated.js#L747) and it looks like `has-background-dim` is the most reliable class to gain specificity here going back to v4. 

Do we need the child combinator (`>`)? For me it has enough weigh to define without it.

```css
	.has-background-dim.has-background-gradient {
		background-color: transparent;
	}
```

Reply:
Thanks for spotting the child combinator issue. Indeed, it is not needed so I went ahead and removed it.